modules/tinymce/src/plugins/media/main/ts/ui/Dialog.ts

```diff
@@ -134,8 +146,21 @@ const handleInsert = (editor: Editor, html: string): void => {
   editor.nodeChanged();
 };
 
+const isEmbedIframe = (type: 'ephox-embed-iri' | 'script' | 'object' | 'iframe' | 'embed' | 'video' | 'audio' | null | undefined, url: string) =>
```

Comment:
Might be worth having something like `export type MediaDataType = 'ephox-embed-iri' | 'script' | 'object' | 'iframe' | 'embed' | 'video' | 'audio';` in the Types.ts file and use that here as well

Reply:
Extracted to a new type `MediaDataType`.